tests/rbac_test.go

```diff
@@ -147,6 +149,34 @@ var _ = Describe("Aggregated role in-action tests", Serial, func() {
 		Entry("[test_id:3949]can do everything with edit", "edit"),
 	)
 
+	DescribeTable("check all user facing resources can be manipulated by non-cluster-admin", func(user string) {
```

Comment:
Maybe pass in a map or list of resources and rbac rules that a user should be allowed to do. This way we are checking all the aggregate roles and not just the admin ones.

Reply:
I think we want to avoid passing a list of resources, and by doing that, we protect the project from introducing unusable new APIs in the future